src/js/components/NestedServiceLinks.js

```diff
@@ -163,9 +151,7 @@ class NestedServiceLinks extends React.Component {
   }
 }
 
-NestedServiceLinks.defaultProps = {
-  taskID: null
-};
+NestedServiceLinks.defaultProps = {};
 
 const classPropType = PropTypes.oneOfType([
```

Comment:
`serviceLink` add to this propType.

Reply:
I think you meant adding to `propTypes` and not `classPropType `